lib/server/index.js

```diff
@@ -330,6 +332,8 @@ module.exports.stop = function () {
 		io.close();
 	}
 
+	require('../replicator').saveAllReplicants();
```

Comment:
Can I have explanation on this line? Is this to ensure that throttled operations are done before server stops?